src/main/java/org/broadinstitute/dsm/DSMServer.java

```diff
@@ -668,4 +685,31 @@ public static boolean isTest(@NonNull String shipperName) {
         }
         return true;
     }
+
+
+    // todo  arz fixme read from config file
+    private static void enableCORS(String methods, String headers) {
```

Comment:
Although there is  already a `protected` version of this method on the superclass, I wanted to add the ability to support multiple URLs.  Browsers were giving me guff about using `*` and `included` token auth, so I addressed it here by echoing back the appropriate URL dynamically.